SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -232,9 +232,19 @@ void FSpatialGDKEditor::GenerateSchema(ESchemaGenerationMethod Method, TFunction
 		else
 		{
 			UE_LOG(LogSpatialGDKEditor, Error, TEXT("Schema generation failed. View earlier log messages for errors."));
+
+			// Make sure SchemaDatabase is not loaded (unlocks the file after any failed schema gen operations)
+			if (UPackage* LoadedDatabase =
+					FindPackage(nullptr, *FPaths::Combine(TEXT("/Game/"), *SpatialConstants::SCHEMA_DATABASE_FILE_PATH)))
```

Comment:
```suggestion
			if (UPackage* LoadedDatabase =
					FindPackage(nullptr, *SpatialConstants::SCHEMA_DATABASE_ASSET_PATH))
```

Reply:
SCHEMA_DATABASE_FILE_PATH is used a bit higher up for the same function (line 149), should that be changed as well?